amxmodx/CTask.cpp

```diff
@@ -269,8 +269,11 @@ bool CTaskMngr::taskExists(int iId, AMX *pAmx)
 
 void CTaskMngr::startFrame()
 {
-	for (auto &task : m_Tasks)
+	auto lastSize = m_Tasks.length();
+	for(auto i = 0; i < lastSize; i++)
```

Comment:
@Arkshine Okey. `lastSize` eliminates the situation when we can have recursive calls. Is it okey ? Like doing `register_event` in `register_event` handler :)